assets/xml/objects/object_skb.xml

```diff
@@ -0,0 +1,64 @@
+        <Limb Name="gStalchildLeftUpperArmLimb" Type="Standard" Offset="0x417C"/>
+        <Limb Name="gStalchildLeftForearmLimb" Type="Standard" Offset="0x4188"/>
+        <Limb Name="gStalchildLeftHandLimb" Type="Standard" Offset="0x4194"/>
+        <Limb Name="gStalchildSpineLimb" Type="Standard" Offset="0x41A0"/>
+
+        <!-- Stalchild DisplayLists -->
+        <DList Name="gStalchildHeadDL" Offset="0x3230"/>
+        <DList Name="gStalchildJawDL" Offset="0x3598"/>
+        <DList Name="gStalchildRibcageDL" Offset="0x3640"/>
+        <DList Name="gStalchildSpineDL" Offset="0x3708"/>
+        <DList Name="gStalchildWaistDL" Offset="0x37A0"/>
+        <DList Name="gStalchildLeftUpperArmDL" Offset="0x38A8"/>
+        <DList Name="gStalchildLeftForearmDL" Offset="0x3968"/>
+        <DList Name="gStalchildLeftHandDL" Offset="0x3A10"/>
+        <DList Name="gStalchildRightUpperArmDL" Offset="0x3AD8"/>
+        <DList Name="gStalchildRightForearmDL" Offset="0x3B98"/>
+        <DList Name="gStalchildRightHandDL" Offset="0x3C40"/>
+        <DList Name="gStalchildLeftFemurDL" Offset="0x3D08"/>
+        <DList Name="gStalchildLeftShinDL" Offset="0x3DA0"/>
+        <DList Name="gStalchildLeftFootDL" Offset="0x3E48"/>
+        <DList Name="gStalchildRightFemurDL" Offset="0x3EE8"/>
+        <DList Name="gStalchildRightShinDL" Offset="0x3F80"/>
+        <DList Name="gStalchildRightFootDL" Offset="0x4028"/>
+
+        <!-- Stalchild Textures -->
+        <Texture Name="gStalchildWaistTex" OutName="stalchild_waist" Format="rgb5a1" Width="16" Height="16" Offset="0x18A0"/>
+        <Texture Name="gStalchildRibCageTex" OutName="stalchild_rib_cage" Format="rgb5a1" Width="8" Height="16" Offset="0x1AA0"/>
+        <Texture Name="gStalchildHeadTex" OutName="stalchild_head" Format="rgb5a1" Width="8" Height="8" Offset="0x1BA0"/>
+        <Texture Name="gStalchildBoneTex" OutName="stalchild_bone" Format="rgb5a1" Width="8" Height="8" Offset="0x1CA0"/>
+        <Texture Name="gStalchildNasalHoleTex" OutName="stalchild_nasal_hole" Format="rgb5a1" Width="8" Height="8" Offset="0x1C20"/>
+        <Texture Name="gStalchildHandAndFeetTex" OutName="stalchild_hand_and_feet" Format="rgb5a1" Width="8" Height="16" Offset="0x1D20"/>
+        <Texture Name="gStalchildTeethTex" OutName="stalchild_teeth" Format="rgb5a1" Width="16" Height="8" Offset="0x1E20"/>
+        <Texture Name="gStalchildEyeSocketTex" OutName="stalchild_eye_socket" Format="rgb5a1" Width="16" Height="16" Offset="0x1F20"/>
+        <Texture Name="gStalchildEyeTex" OutName="stalchild_eye" Format="rgb5a1" Width="8" Height="8" Offset="0x2120"/>
+
+        <!-- Stalchild animations -->
+        <Animation Name="gStalchildBurrowAnim" Offset="0x1854"/>
+        <Animation Name="gStalchildDieAnim" Offset="0x9DC"/>
+        <Animation Name="gStalchildDamageAnim" Offset="0xD98"/>
+        <Animation Name="gStalchildWalkAnim" Offset="0x47E0"/>
+        <Animation Name="gStalchildAttackAnim" Offset="0x460"/>
```

Comment:
I think our convention is to use the gerund rather than the root form for animations, so "Burrowing", "Dying", "Damaged" (because it's passive) etc.